packages/screens/Mini/Chat/components/ChatList.tsx

```diff
@@ -77,7 +77,15 @@ export const ChatList = () => {
         />
         <SpacerColumn size={3} />
         <FlatList
-          data={searchResults}
+          data={searchResults.filter((result) => {
+            if (
+              result?.type === "group" &&
+              (!result?.members || result?.members.filter(Boolean).length === 0) // Hiding conversation from left member of the group
```

Comment:
Wow, quite complex logic

Reply:
when user leave group,  got members array with single 'undefined' element so filtered it out.

Reply:
better fix at root of problem

Reply:
ok